src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -287,6 +288,24 @@ def load_categorical_paf(key: str, location: str) -> pd.DataFrame:
     return paf
 
 
+def load_wasting_transition_rates(key: str, location: str) -> pd.DataFrame:
```

Comment:
I don't think I understand what we're doing with these rates. Are we not still using the complicated system of equations to calculate these transition rates?

Reply:
Correct! From the docs: "Notably, the values for these transition rates are now directly provided rather than only providing equations to calculate the transition rates."